### DISCUSSION TITLE
Ability on mobile to open the overview panel as a new browser tab

### DISCUSSION THREAD

**Discussion by chilliger** [uBlockOrigin/uBlock-issues#2461] [Ideas]
As described [here](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface) on desktop it is possible to open the overview panel with **ctrl** **shift** and clicking on **all** in a new browser tab.

A button on mobile would be nice so making a website work would have a better work flow instead of always going menu -> add-on -> ublock

**Comment by u-RraaLL** [comment]
It's technically possible to manually open it in a new page, but you'd need to know the tab id. It's easier to open the logger and use the popup there.

**Reply by chilliger** [reply]
Where is the logger in firefox android?
<img src="https://user-images.githubusercontent.com/2310896/213868179-45db1cf9-e996-4bc5-b6e5-b43835b5e450.png" width="300">

**Reply by ghost** [reply]
> Where is the logger in firefox android?

Hidden https://github.com/uBlockOrigin/uBlock-issues/wiki/Quick-guide:-popup-user-interface#open-the-logger

**Reply by chilliger** [reply]
The panel in the logger is pretty buggy. Taps do not click where you have put your finger on. Unusable to me.

**Reply by u-RraaLL** [reply]
> The panel in the logger is pretty buggy. Taps do not click where you have put your finger on. Unusable to me.

Works correctly on my side. You can also zoom in if you think the panel is too small.
